index.html

```diff
@@ -8,20 +8,32 @@
     <title>Awesome Books</title>
 </head>
 <body>
+        <nav>
+            <img src="Images/libro-nav" alt="logo">
+            <ul class="menu">
+                <li id="table-books">Books</li>
+                <li id="add-books">Add Books</li>
+                <li id="contact-section">Contact</li>
+            </ul>
+        </nav>
+        <div class="date-time">
+            <!-- <div class="date">
+                <span id="days">Day</span>,
+                <span id="monthly">Month</span>
+                <span id="daynumb">00</span>,
+                <span id="year">Year</span>
+            </div>
+            <div class="times">
+                <span id="hours">00</span><span> :</span>
+                <span id="minutes">00</span><span> :</span>
+                <span id="seconds">00</span>
+                <span id="ampm">AM</span>
+            </div> -->
```

Comment:
- [ ] Let's remove unused pieces of code as this should not be committed to GitHub, instead, we can use GitHub history anytime if we need to review a previous version of our code. [This is considered a best practice, you can read more about it here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)